.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "submodules/privacy-grade"]
 	path = submodules/privacy-grade
 	url = https://github.com/duckduckgo/privacy-grade
+[submodule "vpn-network/vpn-network-impl/src/main/cpp/netguard"]
+	path = vpn-network/vpn-network-impl/src/main/cpp/netguard
+	url = https://github.com/duckduckgo/netguard.git
```

Comment:
This is a submodule for now. I will work in making it a releasable versioned aar in the near future